docs/_documentations/codewind-minikube-guickguide.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: guide
+title: Codewind in Minikube 
+categories: guides 
+guide_description: End-to-end instructions on how to deploy Codewind and projects on local Kube, specifically, Minikube. 
+permalink: codewind-minikube-quick-guide
+duration: 5 minutes
+tags: Codewind, Minikube, microservice
+---
+
+# Codewind in Minikube 
+
+## Objectives 
+* Install Minikube 
+* Install Codewind 
+* Develop a microservice with Codewind on Minikube 
+
+## Overview
+Minikube is essentially Kubernetes but on your local computer. Minikube supports the latest Kubernetes release, so you can configure Minikube on any container runtime. With Minikube, you can start your computer then utilize all Kubernetes' capabilities. Use Minikube to explore and learn to develop with Codewind. This quick guide assists and instructs you to deploy Codewind and projects on Minikube. 
+
+* [Prerequisistes](##prerequisites)
+* [Installing Minikube](##installing-minikube)
+* [Installing Codewind](##installing-codewind)
+* [Developing a microservice](##developing-a-microservice)
+* [What you have learned](##what-you-have-learned)
+* [Next steps](##next-steps) 
+
+## Prerequisites
+* A Mac, Windows, or Linux host.
+* Install [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+* At least 8Gb RAM. Minikube needs at least 4Gb of RAM for this quick guide. 
+
+## Installing Minikube
+1. To run the needed commands, open your terminal. 
+2. To start Minikube on your machine, run, `minikube start --memory=4096`.
+    * Minikube creates a single-node Kubernetes cluster with 4GB of RAM available to it.
+    * If this command fails, refer to [Minikube installation docs](https://kubernetes.io/docs/tasks/tools/install-minikube) and ensure you have enough memory on your system. 
+3. To install [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) on Minikube, run, `minikube addons enable ingress`. 
+    * Codewind makes use of Ingress to expose its endpoints.
+4. To determine your Ingress IP, run `minikube ip`. 
+    * Example:
+     ```
+    $ minikube ip
+    192.168.64.46
+     ```
+     * Save this value for later.
+ 
+## Installing Codewind
+1. Choose your IDE and install the Codewind extension. 
+    * Follow the install instructions for [Eclipse](https://www.eclipse.org/codewind/mdt-eclipse-getting-started.html) or [VS Code](https://www.eclipse.org/codewind/mdt-vsc-getting-started.html). 
+2. Locate the `cwctl` command line interface (CLI). 
```

Comment:
let's use the Codewind operator instructions instead of `cwctl` instructions. Let's simplify the Codewind operator steps include them here in lieu of `cwctl`. 

If we want to link back to the docs, we can link users here: https://www.eclipse.org/codewind/remote-deploying-codewind.html